CMS/templates/partials/footer.html

```diff
@@ -16,6 +16,7 @@
                     <li><a href="https://campaignresources.phe.gov.uk/resources/faqs" class="navigation-link">FAQs</a></li>
                     <li><a href="https://campaignresources.phe.gov.uk/resources/contact_us" class="navigation-link">Contact us</a></li>
                     <li><a href="https://campaignresources.phe.gov.uk/resources/cookie-declaration" class="navigation-link">Cookie declaration</a></li>
+                    <li><a href="{{ accessibility_statement_url }}" class="navigation-link">Accessibility statement</a></li>
```

Comment:
let's make it so that if there is no accessibility statement this link doesn't show

it will need to be something like 
```
{% if accessibility_statement_url %}
<li><a href="{{ accessibility_statement_url }}" class="navigation-link">Accessibility statement</a></li>
{% endif %}
```
And you'll need to make sure accessibility_statement_url returns None if there is no accessibility page